test/e2e/suite/command/push.go

```diff
@@ -102,6 +102,16 @@ var _ = Describe("ORAS beginners:", func() {
 				MatchErrKeyWords("Error:", invalidFlag, "Available options: v1.1, v1.0").
 				Exec()
 		})
+
+		It("should fail if image spec v1.1 is used, with --config and without --artifactType", func() {
+			testRepo := attachTestRepo("test-repo")
```

Comment:
Wrong utility method used. Also the repo name should be related to the test case, e.g. `v1-1/no-artifact-type`
```suggestion
			testRepo := pushTestRepo("test-repo")
```